### ISSUE TITLE
Matplotlib import issue with nilearn 0.4.0 on PyPi

### ISSUE THREAD

**Issue by ljchang** [nilearn/nilearn#1618]
We have been having an issue with the current version of nilearn on PyPi (0.4.0).

We receive the following error when running our test suite:

```
../../anaconda3/lib/python3.6/site-packages/nilearn/plotting/img_plotting.py:511: in <module>
    cmap=plt.cm.spectral, vmin=None, vmax=None, **kwargs):
E   AttributeError: module 'matplotlib.cm' has no attribute 'spectral'
```

On the current github version, the line has been updated to:
```
             cmap=plt.cm.nipy_spectral, vmin=None, vmax=None, **kwargs):

```

Just wanted to let people know that they should use the current version on github if they are running into this error.  `pip install git+https://github.com/nilearn/nilearn`

**Comment by GaelVaroquaux** [comment]
This was fixed by #1616 and will be included in next release.

**Comment by KamalakerDadi** [comment]
@ljchang We now have a new release. Hope there shouldn't be any problem now.
